lib/Test/PostgreSQL.pm

```diff
@@ -116,7 +116,42 @@ has socket_dir => (
   is => "ro",
   isa => Str,
   lazy => 1,
-  default => method () { File::Spec->catdir( $self->base_dir, 'tmp' ) },
+  default => method () {
+    my $dir = File::Spec->catdir( $self->base_dir, 'tmp' );
```

Comment:
This anonymous default method has ended up quite long now -- if it's easy, could it get refactored out to a named method and just referenced from here? It might help readability.

Reply:
Yeah I was thinking about that when I was working on this PR... In the end decided against splitting it out since the length of the method is basically all in the comments, there's not that much code in it. If we split it out, that's going to be just another long function, although a named one.